src/java.base/share/classes/java/lang/StringBuilder.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If there's no change to this file the copyright year update could be reverted?

Reply:
Good catch, Daniel! Fixed. Thank you! 🙂